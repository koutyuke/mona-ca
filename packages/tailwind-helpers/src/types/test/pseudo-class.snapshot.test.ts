import { describe, expect, test } from "bun:test";
import { readFileSync } from "node:fs";
import { resolve } from "node:path";

describe("pseudo-class snapshot", () => {
	test("should match snapshot", () => {
		const filePath = resolve(__dirname, "../pseudo-class.type.ts");
		const fileContent = readFileSync(filePath, "utf-8");

		expect(fileContent).toMatchSnapshot();
	});
});
