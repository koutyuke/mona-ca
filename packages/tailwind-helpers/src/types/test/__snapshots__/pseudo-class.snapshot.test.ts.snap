// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`pseudo-class snapshot should match snapshot 1`] = `
"export type PseudoClass =
	| "*"
	| "first-letter"
	| "first-line"
	| "marker"
	| "selection"
	| "file"
	| "placeholder"
	| "backdrop"
	| "before"
	| "after"
	| "first"
	| "last"
	| "only"
	| "odd"
	| "even"
	| "first-of-type"
	| "last-of-type"
	| "only-of-type"
	| "visited"
	| "target"
	| "open"
	| "default"
	| "checked"
	| "indeterminate"
	| "placeholder-shown"
	| "autofill"
	| "optional"
	| "required"
	| "valid"
	| "invalid"
	| "in-range"
	| "out-of-range"
	| "read-only"
	| "empty"
	| "focus-within"
	| "hover"
	| "focus"
	| "focus-visible"
	| "active"
	| "enabled"
	| "disabled"
	| "group-first"
	| "group-last"
	| "group-only"
	| "group-odd"
	| "group-even"
	| "group-first-of-type"
	| "group-last-of-type"
	| "group-only-of-type"
	| "group-visited"
	| "group-target"
	| "group-open"
	| "group-default"
	| "group-checked"
	| "group-indeterminate"
	| "group-placeholder-shown"
	| "group-autofill"
	| "group-optional"
	| "group-required"
	| "group-valid"
	| "group-invalid"
	| "group-in-range"
	| "group-out-of-range"
	| "group-read-only"
	| "group-empty"
	| "group-focus-within"
	| "group-hover"
	| "group-focus"
	| "group-focus-visible"
	| "group-active"
	| "group-enabled"
	| "group-disabled"
	| "group"
	| "peer-first"
	| "peer-last"
	| "peer-only"
	| "peer-odd"
	| "peer-even"
	| "peer-first-of-type"
	| "peer-last-of-type"
	| "peer-only-of-type"
	| "peer-visited"
	| "peer-target"
	| "peer-open"
	| "peer-default"
	| "peer-checked"
	| "peer-indeterminate"
	| "peer-placeholder-shown"
	| "peer-autofill"
	| "peer-optional"
	| "peer-required"
	| "peer-valid"
	| "peer-invalid"
	| "peer-in-range"
	| "peer-out-of-range"
	| "peer-read-only"
	| "peer-empty"
	| "peer-focus-within"
	| "peer-hover"
	| "peer-focus"
	| "peer-focus-visible"
	| "peer-active"
	| "peer-enabled"
	| "peer-disabled"
	| "peer"
	| "has"
	| "group-has"
	| "peer-has"
	| "aria-busy"
	| "aria-checked"
	| "aria-disabled"
	| "aria-expanded"
	| "aria-hidden"
	| "aria-pressed"
	| "aria-readonly"
	| "aria-required"
	| "aria-selected"
	| "aria"
	| "group-aria-busy"
	| "group-aria-checked"
	| "group-aria-disabled"
	| "group-aria-expanded"
	| "group-aria-hidden"
	| "group-aria-pressed"
	| "group-aria-readonly"
	| "group-aria-required"
	| "group-aria-selected"
	| "group-aria"
	| "peer-aria-busy"
	| "peer-aria-checked"
	| "peer-aria-disabled"
	| "peer-aria-expanded"
	| "peer-aria-hidden"
	| "peer-aria-pressed"
	| "peer-aria-readonly"
	| "peer-aria-required"
	| "peer-aria-selected"
	| "peer-aria"
	| "data"
	| "group-data"
	| "peer-data"
	| "supports"
	| "motion-safe"
	| "motion-reduce"
	| "contrast-more"
	| "contrast-less"
	| "max-tablet"
	| "max-laptop"
	| "max-desktop"
	| "max"
	| "tablet"
	| "laptop"
	| "desktop"
	| "min"
	| "portrait"
	| "landscape"
	| "ltr"
	| "rtl"
	| "dark"
	| "forced-colors"
	| "print";
"
`;
